scanners/android/android.go

```diff
@@ -91,16 +91,109 @@ func (scanner *Scanner) Options() (models.OptionNode, models.Warnings, models.Ic
 
 // DefaultOptions ...
 func (scanner *Scanner) DefaultOptions() models.OptionNode {
-	projectLocationOption := models.NewOption(ProjectLocationInputTitle, ProjectLocationInputEnvKey)
-	moduleOption := models.NewOption(ModuleInputTitle, ModuleInputEnvKey)
-	variantOption := models.NewOption(VariantInputTitle, VariantInputEnvKey)
-	configOption := models.NewConfigOption(DefaultConfigName, nil)
+	// projectLocationOption := models.NewOption(ProjectLocationInputTitle, ProjectLocationInputEnvKey).SetType(models.TypeUserInput)
```

Comment:
please revert these changes and add the option type input to the option initialiser function